### ISSUE TITLE
Add documentation for solving rules conflicts

### ISSUE THREAD

**Issue by mk-pmb** [zetlen/eslint-plugin-package-json#4]
Hi!
Turns out a lot of my problems from https://github.com/zetlen/eslint-plugin-package-json/pull/3 already exist in the master branch:
[Test branch with minimal rules](https://github.com/mk-pmb/try-eslint-plugin-package-json-privacy-mgmt-js/tree/comma-dangle-quotes-semi) → [Travis report](https://travis-ci.org/mk-pmb/try-eslint-plugin-package-json-privacy-mgmt-js/jobs/572532710)
→ Offending rules:

```yaml
    comma-dangle: ["error", {"objects": "always-multiline"}] # +4 errors
    quotes: ["error", "single"] # +33 errors
    semi: ["error", "never"] # +1 error
```

Could you add documentation for how to resolve conflicts with these rules? Probably in "Usage" in the readme.

**Comment by zetlen** [comment]
@mk-pmb Unless I'm misunderstanding, this issue has more to do with ESLint proper than with package.json specifically. If two rules conflict, it's best not to run them on the same document, and you can [use override configurations](https://eslint.org/docs/user-guide/configuring) to do that.

I do need to add some better docs about how to make eslint run on package.json files in the first place.

If I'm misunderstanding your concern, please reopen the issue!
